features/step_definitions/api_request_steps.rb

```diff
@@ -243,6 +243,16 @@
   }]
 end
 
+Given("I add {string} of multiple regular_transactions, of {float} per month") do |category, amount|
```

Comment:
can we include debit in the name of this step

Reply:
Yep will change